File: Rust/GrasshoperGradeBook/MySolution.rs
fn get_grade(s1: u16, s2: u16, s3: u16) -> char {
    let letterGrade = (s1+s2+s3)/3;
    match letterGrade {
        0..=59 => 'F',
        60..=69 => 'D',
        70..=79 => 'C',
        80..=89 => 'B',
        _ => 'A',
    }
}

// Original kata: https://www.codewars.com/kata/55cbd4ba903825f7970000f5
// My solution: https://www.codewars.com/kata/reviews/631c050f15064f0001a55e97/groups/6799dfbbf8efc2f4023a2b08

File: PHP/CreatePhoneNumber/MySolution.php
<?php
function createPhoneNumber($numbersArray) {
  return "($numbersArray[0]$numbersArray[1]$numbersArray[2]) $numbersArray[3]$numbersArray[4]$numbersArray[5]-$numbersArray[6]$numbersArray[7]$numbersArray[8]$numbersArray[9]";
}

// Original kata: https://www.codewars.com/kata/525f50e3b73515a6db000b83
// My solution: https://www.codewars.com/kata/reviews/5dc7af8d2d55c300012ecfdf/groups/5dd5592f0466110001a7e375
